invenio_communities/assets/semantic-ui/js/invenio_communities/members/components/ErrorMessage.js

```diff
@@ -0,0 +1,41 @@
+/*
+ * This file is part of Invenio.
+ * Copyright (C) 2022 CERN.
+ *
+ * Invenio is free software; you can redistribute it and/or modify it
+ * under the terms of the MIT License; see LICENSE file for more details.
+ */
+
+import React from "react";
+import { Message, List } from "semantic-ui-react";
+import { communityErrorSerializer } from "../../api/serializers";
+import _flatten from "lodash/flatten";
+
+export const ErrorMessage = ({ error }) => {
+  let { message, errors } = communityErrorSerializer(error);
+
+  const noErrorResponse = !message && !errors;
+
+  if (noErrorResponse) {
+    message = error.message;
+  }
+
+  const getAllErrorMessages = (errors) => {
+    const messages = errors.map((error) => error.messages);
+
+    return _flatten(messages);
+  };
+
+  return (
+    <Message error>
+      {message && message}
+      {errors && (
+        <List bulleted>
```

Comment:
there is a shorthand for the list https://react.semantic-ui.com/collections/message/#variations-error